imap_processing/hi/l1a/histogram.py

```diff
@@ -92,13 +90,16 @@ def allocate_histogram_dataset(num_packets: int) -> xr.Dataset:
     dataset : xarray.Dataset
         Empty xarray.Dataset ready to be filled with packet data.
     """
+    cdf_manager = CdfAttributeManager(imap_module_directory / "cdf" / "config")
+    cdf_manager.load_global_attributes("imap_hi_global_cdf_attrs.yaml")
+    cdf_manager.load_variable_attributes("imap_hi_variable_attrs.yaml")
```

Comment:
@maxinelasp What is main difference between doing this way and other way I have seen others do? Is there benefit of using one over other? Other way I have seen others do is this way:
```
    # create the attribute manager for this data level
    attr_mgr = ImapCdfAttributes()
    attr_mgr.add_instrument_global_attrs(instrument="lo")
    attr_mgr.add_instrument_variable_attrs(instrument="lo", level="l1a")
    attr_mgr.add_global_attribute("Data_version", data_version)
```

Reply:
I can answer that...
Using `ImapCdfAttributes` adds the automatic setting of the config directory as part of the constructor. It also adds convenience functions for adding global and variable attributes by instrument name. For some reason, I thought that would preclude me from using the variable attribute YAML that I have which lumps all l1 variable attributes into a single file so that I can leverage the YAML inheritance and avoid duplicate entries across separate `l1a`, `l1b`, `l1c`. Now that I am thinking a bit more clearly, I should probably do the following:
```suggestion
    attr_mgr = ImapCdfAttributes()
    attr_mgr.add_instrument_global_attrs(instrument="lo")
    attr_mgr.load_variable_attributes("imap_hi_variable_attrs.yaml")
```

I wrote up a feature request to add inheritance across yaml files: #664
This would allow me to follow what other instruments are doing. I just find it very useful to be able to define defaults for each dtype variable and have consistent, not repeated definitions for variables like `epoch`, `ccsds_met`, and any other variables that have the same definition across products.